upup/models/cloudup/resources/addons/karpenter.sh/k8s-1.19.yaml.template

```diff
@@ -639,286 +694,295 @@ data:
   "batchMaxDuration": "10s"
   "batchIdleDuration": "1s"
 ---
-# Source: karpenter/templates/controller/rbac.yaml
+# Source: karpenter/templates/configmap.yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: karpenter-global-settings
+  namespace: kube-system
+data:  
+    "aws.clusterEndpoint": "https://{{ APIInternalName }}"
+    "aws.clusterName": "{{ ClusterName }}"
+    "aws.defaultInstanceProfile": ""
+    {{ if not .Networking.AmazonVPC }}
+    "aws.enableENILimitedPodDensity": "true"
+    {{ else }}
+    "aws.enableENILimitedPodDensity": "false"
+    {{ end }}
+    "aws.enablePodENI": "false"
+    "aws.interruptionQueueName": "{{ ClusterName }}"
+    "aws.isolatedVPC": "false"
+    {{ if UsesInstanceIDForNodeName }}
+    "aws.nodeNameConvention": "resource-name"
+    {{ else }}
+    "aws.nodeNameConvention": "ip-name"
+    {{ end }}
+    "aws.vmMemoryOverheadPercent": "0.075"
+    "batchIdleDuration": "1s"
+    "batchMaxDuration": "10s"
+---
+# Source: karpenter/templates/clusterrole.yaml
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: karpenter-controller
```

Comment:
Since this PR changes many of the resource names, pruning must be enabled to clean up old resources: https://github.com/kubernetes/kops/pull/14601/files#diff-13f16294f2d4be71ad5d7207a320b919ec0eb217c5be43a7ee0c91e3b17e1f99R1205

Reply:
Pruning enabled :ok_hand: 
 - https://github.com/kubernetes/kops/pull/15144/commits/096eb62c33fa2574fc70ed86fef96ba406480e40